src/components/CookiesBanner/index.tsx

```diff
@@ -114,7 +114,7 @@ const CookiesBannerForm = (props: {
       <div className={classes.content}>
         {key && (
           <div className={classes.intercomAlert}>
-            <img src={AlertRedIcon} />
+            <img src={AlertRedIcon} alt="" />
```

Comment:
If you are adding an `alt` attribute, make sure to alternative text. That's what it's for.

Reply:
For decorative images, the alt attribute [should be empty](https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link) since it doesn't add any additional information. But it also needs to exist (lint)